src/ui_framework/README.md

```diff
@@ -7,9 +7,39 @@
 
 ## What is this?
 
-The UI Framework provides you with UI components you can quickly use to build UIs, as well as interactive examples which document how they're supposed to be used. These UI components are currently only implemented in CSS and markup, but eventually they'll grow to involve JS as well.
+The Kibana UI Framework provides you with UI components you can quickly use to build user interfaces for Kibana.
 
-When you build a UI using this framework (e.g. a plugin's UI), you can rest assured it will fit into the overall Kibana UI.
+The UI Framework comes with interactive examples which document how to use its various UI components. These components are currently only implemented in CSS, but eventually they'll grow to involve JS as well.
+
+When you build a UI using this framework (e.g. a plugin's UI), you can rest assured it will integrate seamlessly into the overall Kibana UI.
+
+## How to create a new component
+
+There are two steps to creating a new component:
+
+1. Create the CSS for the component in `ui_framework/components`.
+2. Document it with examples in `ui_framework/doc_site`.
+
+### Create the component CSS
+
+1. Create a directory for your component in `ui_framework/components`.
+2. Create an `_index.scss` file and however many component SCSS files you need in this directory. Import the component SCSS files into `_index.scss`.
```

Comment:
The `_index.scss` file is used to import all of the components so that only this index file needs to be imported in the root `index.scss` file. It's just a way to keep your files organized.

I just updated the README with more detail on how multiple components can be organized together in a single directory.